cfb-mode/src/lib.rs

```diff
@@ -51,12 +51,20 @@
 pub extern crate stream_cipher;
 extern crate block_cipher_trait;
 
+#[cfg(cargo_feature = "zeroize")]
```

Comment:
AFAICT `#[cfg(cargo_feature = "zeroize")]` does absolutely nothing. I think all of these need to be changed to `#[cfg(feature = "zeroize")]`